torch/ao/quantization/pt2e/quantizer/x86_inductor_quantizer.py

```diff
@@ -28,18 +29,94 @@
     PerChannelMinMaxObserver,
 )
 from torch.ao.quantization.qconfig import _ObserverOrFakeQuantizeConstructor
-from typing import List, Dict, Optional, Set, Any
+from typing import List, Dict, Optional, Set, Any, Tuple
 from torch.fx import Node
 from torch.fx.passes.utils.source_matcher_utils import (
     get_source_partitions,
     SourcePartition,
 )
+from dataclasses import dataclass
 
 __all__ = [
     "X86InductorQuantizer",
     "get_default_x86_inductor_quantization_config",
 ]
 
+@dataclass
+class X86InductorQuantizationAnnotation(QuantizationAnnotation):
+    # _is_output_of_fusion_pattern:
+    #  * Node as output node of a fusion pattern.
+    _is_output_of_fusion_pattern: bool = False
+    # _is_quantized_single_node:
+    #  * Node doesn't belong to any fusion pattern.
+    #  * Node among the quantizable_ops_pt2e set.
+    #  * Node has fake quant inserted at all of the inputs.
+    _is_quantized_single_node: bool = False
+
+# Ops support int8 data type and excludes ops like conv, linear.
+quantizable_ops_pt2e: Set = {
+    torch.ops.aten.max_pool2d_with_indices.default,
+}
+
+# Ops that:
+# 1. Ops prefer to run with int8 when int8 input is given.
+# 2. Ops don't support int8 in and fp32 out.
+int8_in_int8_out_ops_pt2e: Set = {
+    torch.ops.aten.max_pool2d_with_indices.default,
+}
+
+# Ops input and output share observer.
+in_out_share_obs_ops_pt2e: Set = {
```

Comment:
Where is it used?

Reply:
It's not used yet. Remove it in this PR.